csss-site/src/events/frosh/static/frosh_static/2023/css/section.css

```diff
@@ -0,0 +1,91 @@
+.section-anchor {
+	position: absolute;
+	z-index: 99;
+	top: -64px;
+	width: 1px;
+	height: 1px;
+}
+section {
+	position: relative;
+	margin: 0 auto;
+	border-bottom: 1px solid var(--gray);
+	padding: 64px max(32px, 50% - 400px);
+	width: min(800px, 100% - 64px);
+	background-color: var(--white);
+	color: var(--black);
+}
+section .spacer {
+	width: 100%;
+	height: 16px;
```

Comment:
I suggest changing this to 24px or 32px, since the paragraphs are a bit claustrophobic